examples/bigcommerce/components/header.tsx

```diff
@@ -26,22 +26,13 @@ export function Header({ className }: Props) {
             </svg>
           </a>
         </Link>
-        <Link href={'/'}>
+        <Link href={'/#featured'}>
```

Comment:
Should this link come from the Makeswift builder directly? I'm assuming the `featured` HTML ID was assigned in the Makeswift builder.

Maybe the header can be added as a component and the links can be a `List` of `Link`s?

Reply:
Done